src/ConstructiveSolidGeometry/VolumePrimitives/Torus.jl

```diff
@@ -98,7 +98,7 @@ function sample(t::Torus{T}, step::Real) where {T}
     ]
 end
 
-function sample(t::Torus{T}, Nsamps::NTuple{3,Int}) where {T}
+function sample(t::Torus{T}, Nsamps::NTuple{3,Int} = (2,5,3)) where {T}
```

Comment:
Why does this need a default value?

Reply:
For sampling additional grid points at the beginning of the electric potential calculation